build.gradle.kts

```diff
@@ -15,10 +15,9 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
 
   implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:2.0.1")
-  implementation("org.springframework.cloud:spring-cloud-starter-aws-messaging:2.2.6.RELEASE")
```

Comment:
The security job was failing: https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-audit-api/1235/workflows/04d84fe4-1a85-436f-9f2a-f444086145f9/jobs/3392

This dependency was pulling in an old version of aws-sdk-java but it doesn't appear to be being used so I've just removed it.